### ISSUE TITLE
Sign Document - Feature Request.

### ISSUE THREAD

**Issue by trb143** [keybase/client#19898]
I am trying to use a Certificate to Sign a PDF document.  I have managed to generate a P12 file and sign the PDF attaching the certificate into the PDF.

When looking at the Certificate is says the certificate has no parent which is correct.

Would it be possible for Keybase to generate a document signing p12 or pem based on the data it holds as it already understands the personal key and has proof of ownership?

The perfect secure solution would be to generate a certificate in memory then sign and attach the certificate to the PDF not returning the certificate.  The certificate would only be available for users who have a proven identity.

The sign API will sign a document but it is not visible to the user when viewing it which does not meet my requirements.

**Comment by taxilian** [comment]
It would be epic if keybase had the capability to sign PDF files =]

**Comment by taxilian** [comment]
Of course, PDF digital signatures use SSL certificates; I'm not sure if keybase has the right type of certs to do that.
